src/app/providers/TableConfigProvider.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, PropsWithChildren, useState } from 'react'
+import {
+  TableConfigContext,
+  TableConfigProviderContext,
+  TableConfigProviderState,
+} from './TableConfigContext'
+import { storage } from '../utils/storage'
+import { StorageKeys } from '../../types/storage'
+import { TableAgeType } from '../../types/table-age-type'
+
+const localStorage = storage()
+
+const tableConfigProviderInitialState: TableConfigProviderState = {
+  ageHeaderType: localStorage.get(StorageKeys.TableAgeType) ?? TableAgeType.Distance,
```

Comment:
This would not hydrate when another window/tab changes the table view. But we can always add https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event